recipes/fastclasses-json/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "fastclasses-json" %}
+{% set version = "0.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fastclasses-json-{{ version }}.tar.gz
+  sha256: bd0edd1d6cb30712b76056f6a9dceb6ce50a6dea1cc760d8720facce8533d975
+
+build:
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - wheel
+    - pip
+  run:
+    - python
+    - typing-extensions  # [py==37]
```

Comment:
You can either build only >=3.8 and remove this dependency, or let typing-extensions get installed on >=3.8. It should not be a problem but that way we could make this one noarch.

Reply:
Good point, it should be noarch now